integreat_cms/cms/forms/pages/page_translation_form.py

```diff
@@ -20,4 +20,4 @@ class Meta:
         #: The model of this :class:`django.forms.ModelForm`
         model = PageTranslation
         #: The fields of the model which should be handled by this form
-        fields = CustomContentModelForm.Meta.fields + ["slug"]
+        fields = CustomContentModelForm.Meta.fields + ["slug", "hix_score"]
```

Comment:
This line currently has no effect, right?
But ok, since we're probably adding this field to the form at some point in the future anyway, I guess it can stay.